src/coreclr/jit/compiler.h

```diff
@@ -9851,7 +9851,8 @@ class Compiler
             // On these platforms we assume the register that the target is
             // passed in is preserved by the validator and take care to get the
             // target from the register for the call (even in debug mode).
-            static_assert_no_msg((RBM_VALIDATE_INDIRECT_CALL_TRASH & (1 << REG_VALIDATE_INDIRECT_CALL_ADDR)) == 0);
+            static_assert_no_msg((RBM_VALIDATE_INDIRECT_CALL_TRASH & regMaskTP(1 << REG_VALIDATE_INDIRECT_CALL_ADDR)) ==
```

Comment:
We can make `IsRegNumInMask` `constexpr` to make this a bit more natural (feel free to do as part of a follow-up).

Reply:
I tried doing that, but as a cascade effect I need to make `genSingleTypeRegSet`, `getRegForType()`, etc. as `constexpr` which I am not sure is worth doing for just an assert. Will pass this on for now.